jscomp/others/js_string.ml

```diff
@@ -34,7 +34,7 @@ external fromCodePointMany : int array -> t = "String.fromCodePoint" [@@bs.val]
 (* String.raw: ES2015, meant to be used with template strings, not directly *)
 
 external length : t -> int = "" [@@bs.get]
-external get : t -> int -> t = "" [@@bs.get_index]
+external get : t -> int -> t option = "" [@@bs.get_index][@@bs.return {undefined_to_opt}]
```

Comment:
As far as I can tell, `[@@bs.return nullable]` is the correct extension point: https://bucklescript.github.io/docs/en/null-undefined-option.html#tips-tricks

Reply:
Yes it is